docker-compose/monitor/docker-compose.yml

```diff
@@ -9,8 +9,6 @@ services:
     environment:
       - METRICS_STORAGE_TYPE=prometheus
       - PROMETHEUS_SERVER_URL=http://prometheus:9090
-      - PROMETHEUS_QUERY_SUPPORT_SPANMETRICS_CONNECTOR=${PROMETHEUS_QUERY_SUPPORT_SPANMETRICS_CONNECTOR:-true}
-      - PROMETHEUS_QUERY_NAMESPACE=${PROMETHEUS_QUERY_NAMESPACE:-}
```

Comment:
@yurishkuro good point! I added this line back in. I don't believe we need to provide the default here however. If its unset then the default will be used. This can be seen by the integration tests passing which don't set this environment variable.